exp-player/addon/components/exp-overview/component.js

```diff
@@ -33,32 +33,38 @@ var generateValidators = function(questions) {
 const questions = [
   {
     question: 'survey.sections.1.questions.1.label',
+    keyName: 'Age',
     type: 'select',
     scale: range(16, 100),
     value: null
   },
   {
     question: 'survey.sections.1.questions.2.label',
+    keyName: 'Gender',
     type: 'select',
-    scale: ['survey.sections.1.questions.2.options.male',
-        'survey.sections.1.questions.2.options.female',
-        'survey.sections.1.questions.2.options.other',
-        'survey.sections.1.questions.2.options.na'
+    scale: [
+        {label: 'survey.sections.1.questions.2.options.male', value: 1},
```

Comment:
Redoing the radio buttons to support separate options/values should make it much easier to control serialization- thanks!